src/antsibull_docs/cli/doc_commands/_build.py

```diff
@@ -165,6 +178,132 @@ def _register_extra_docs(
             output.register_pattern(directory, "*")
 
 
+def _compose_redirect_sentence(
+    collection: str,
+    new_name: str,
+    redirect_replacement_major_version: int,
+    current_major_version: int | None,
+) -> str:
+    if (
+        current_major_version is not None
+        and current_major_version == redirect_replacement_major_version
+    ):
+        return (
+            f"The content of {collection} have been replaced"
```

Comment:
```suggestion
            f"The content of {collection} has been replaced"
```

Reply:
We could go two ways here:
"The contents of foo have been"
or
"The content of foo has been"

Given that other text uses 'content' frequently, I recommend the suggested edit here and on other lines like this.

Reply:
I changed it to `content ... has been` in 167496b14485edb44cc585dbc865bf01e7a6d788.